data/transactions/logic/eval_test.go

```diff
@@ -941,6 +945,7 @@ func TestGlobal(t *testing.T) {
 			CreatorAddress, globalV5TestProgram,
 			EvalStateful, CheckStateful,
 		},
+		6: {GroupID, globalV5TestProgram, Eval, Check},
```

Comment:
I'm not sure why does it work, this assumes assembler version 6 but the field defined as version 5 constant
```
{GroupID, StackBytes, modeAny, 5},
```
This means the test is never called. I'll submit a PR checking `len(tests) <= AssemblerMaxVersion`